install/linux/usr/share/odemis/sim/sparc2-ded-focus-test-sim.odm.yaml

```diff
@@ -0,0 +1,337 @@
+# Configuration for the SPARC v2 system with the 2nd spectrometer (connected via optical fiber from the plate 2)
```

Comment:
Please update the comments at the beginning of each file to explain what's special with these two microscope files. Just looking at them quickly, it's hard to understand why you need two of them, and what's different between them. Is the only difference the focus being associated to a spectrograph instead of another one?
If so, I think it should be possible to have 2 of them, and give a new role to the second focus. I'd suggest "spec-ded-focus".

Reply:
Thanks for clarifying. Now, I understand that they are very similar, but you need two, to test the autofocus on the 2 cameras. I've discussed with Sabrina about changing the SimCam to not provide its own focus actuator, but just support getting it from a separate actuator. If this is done, we should be able to merge the two simulator files together (as both cameras could follow the same focus).

Reply:
Just pushed a PR for this feature. So you should be able to use it very soon :-)